Python/compile.c

```diff
@@ -5699,6 +5708,10 @@ compute_code_flags(struct compiler *c)
     /* (Only) inherit compilerflags in PyCF_MASK */
     flags |= (c->c_flags->cf_flags & PyCF_MASK);
 
+    if ((c->c_flags->cf_flags & PyCF_ALLOW_TOP_LEVEL_AWAIT) && c->u->u_ste->ste_coroutine) {
+        flags |= CO_COROUTINE;
```

Comment:
This thing has some weird effect and turn async_generators into coroutine;  I'm going to change that to `!ste->ste_generator`

Reply:
Please fix to fit 79 characters

Reply:
Done.